main.tf

```diff
@@ -34,6 +34,7 @@ resource "google_compute_subnetwork" "subnetwork" {
   ip_cidr_range            = "${lookup(var.subnets[count.index], "subnet_ip")}"
   region                   = "${lookup(var.subnets[count.index], "subnet_region")}"
   private_ip_google_access = "${lookup(var.subnets[count.index], "subnet_private_access")}"
+  enable_flow_logs         = "${lookup(var.subnets[count.index], "subnet_flow_logs")}"
```

Comment:
@andreyk-code We should add a default (`false`) here so it's not required to pass. Also should document this in the README.

Reply:
Good point! Added in latest commit and updated README. Also made the same change for "private_ip_google_access" value.